Makefile

```diff
@@ -90,7 +90,7 @@ endif
 #######################################################
 
 # hugo version check
-HUGO_VERSION_MIN=114
+HUGO_VERSION_MIN=126
```

Comment:
Should we just make the minimum 140.1 since that's what we tested with?

Reply:
Hmm, I would keep it as is, since it indicates when the latest feature that we’re using was introduced. Newer Hugo releases can also occasionally introduce breaking changes, but we don't check for a "maximum" version.